database-service/scripts/waitForService.sh

```diff
@@ -9,8 +9,8 @@ wait_for_port() {
     for i in `seq 1 $HASURA_GRAPHQL_MIGRATIONS_SERVER_TIMEOUT`;
     do
         echo "waiting for graphql engine"
-        nc -z $HOST $PORT
-        [ "$?" == "0" ] && return || sleep 1
+        version=$(curl -s http://$HOST:$PORT/v1/version | jq ".version")
+        [ "$version" == "" ] && sleep 1 || return
```

Comment:
The previous check were still failing sometimes. Therefore I improved it that way.